config/settings.d/httpboot.yml.example

```diff
@@ -0,0 +1,10 @@
+---
+# Enable publishing of a given directory under /EFI and /httpboot paths.
+# Directory listing is not possible, symlinks are followed but not outside
+# of the rood directory specified in this file.
+
+# Enables the module, make sure to enable TFTP module as well to allow
```

Comment:
You can use `requires` in the plugin spec to make sure the tftp module is loaded.